recipes-core/systemd/files/20-network-io.conf

```diff
@@ -1,7 +1,8 @@
-net.core.wmem_default=524288
 net.core.rmem_default=524288
-net.core.optmem_max=65536
+net.core.rmem_max=8388608
+net.core.wmem_default=524288
 net.core.wmem_max=8388608
+net.core.optmem_max=65536
```

Comment:
The change to this value is not documented in the commit message.

Reply:
Same as above, simply ordering changed